libraries/datadogstatsd.php

```diff
@@ -139,6 +139,23 @@ public static function updateStats($stats, $delta = 1, $sampleRate = 1.0, $tags
         static::send($data, $sampleRate, $tags);
     }
 
+    private static function serialize_tags($tags) {
+        if (is_array($tags) && count($tags) > 0) {
+            $data = array();
+            foreach ($tags as $tag_key => $tag_val) {
+                if (isset($tag_val)) {
+                    array_push($data, $tag_key . ':' . $tag_val);
+                } else {
+                    array_push($data, $tag_key);
+                }
+            }
+            return "|#".implode(",", $data);
+        } elseif (!empty($tags)) {
+            return '|#' . $tags;
```

Comment:
nit: could you use double quote or single quote everywhere for strings for consistency?

Reply:
Good catch !